### DISCUSSION TITLE
API & Webhooks: March Community Check-In 🌱

### DISCUSSION THREAD

**Discussion by ghostinhershell** [community/community#49007] [Apps, API and Webhooks]
Hey folks, reporting in for our March API and Webhooks Community Check-In ✅

## 🧠​​ Pros in the Community
**Big kudos to these superstar community contributors 👏**
- We’ve got another self-serve win this time around. Shout out to @BruceMacD for figuring out why [a SSH key added for a user could not access the organizations that the user belonged to](https://github.com/orgs/community/discussions/48323).
- Got a couple of wins this week for @jge162 🏆
  - Big thanks for helping a community member figure out [how to query the Github API using gh CLI for a PRs with specific keyword in title](https://github.com/orgs/community/discussions/47330).
  - Here’s another Community thank you for teaching community members that [the GitHub API does not provide the total lines of code (LOC) for a specific commit in a repository](https://github.com/orgs/community/discussions/45955). 

Remember to thank these and other community members that take the time to comment and help troubleshoot these problems. An acknowledgement of a job well done can have an incredible effect and make someone’s day. ☀️ We’re here to learn, discover and connect so try and mark a comment as an answer if suitable, give a thank you and select a reaction as a way to show your appreciation! 

For community members who post questions, marking responses that solves your question as the answer goes a long way. When you mark a comment in the Discussion as an answer, GitHub will highlight the comment and replies to the comment to help other community members quickly find the answer as well.

You can learn more about marking a response as the answer to your Community post [here](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer).  

## 🗞️What’s New?
**Updates that were made in the Ecosystems Space 🪐** 
- **[API requests are available via audit log streaming – Private Beta](https://github.blog/changelog/2023-02-01-api-requests-are-available-via-audit-log-streaming-private-beta/)** - GitHub Enterprise Cloud customers can now join a private beta which allows API request events to be streamed as part of their enterprise audit log.
- **[API for reverting a pull request](https://github.blog/changelog/2023-01-27-api-for-reverting-a-pull-request/)** - A GraphQL mutation is now available for reverting a merged pull request: `[revertPullRequest](https://docs.github.com/graphql/reference/mutations#revertpullrequest)`.
- **[Remove Enterprise Members Available via API](https://github.blog/changelog/2023-01-30-remove-enterprise-members-via-api/)** - GitHub Enterprise owners can now leverage the `removeEnterpriseMember API` to remove enterprise members from their GitHub Enterprise.

## 💡Inspiration Station
**Fresh content to give you new ideas!**
- **New to GitHub?** Check out @[potaders](https://github.com/potaders) Community post **[Getting Started with GitHub Part 2: Best practices for notifications](https://github.com/orgs/community/discussions/47569)**.
  - **Missed Part 1?** Check that out here: **[Getting Started with GitHub Part 1: Best practices for setting up your GitHub account](https://github.com/orgs/community/discussions/39082)**
- **Questions about authenticating as a GitHub App?** Check out @loujr’s Community post [GitHub App Installation Token and Authenticating as a GitHub App](https://github.com/orgs/community/discussions/48186) 🔥

## 📖 Open Source Resources
**New to open source?** Here are some resources to help you get started:
- **Did you hear?** GitHub is now home to 100 million developers and counting 🎉Here’s a look at [Open Source’s Impact on the World’s 100 Million Developers](https://github.blog/2023-02-01-open-sources-impact-on-the-worlds-100-million-developers/).
- **Want to start contributing to open source projects?** [Here’s how to find projects that need help so you can start making impactful contributions today](https://github.com/collections/choosing-projects) 🏗️
- **Did you know that [GitHub’s Docs](https://docs.github.com/en)** are open source?! 📖Interested in contributing to GitHub’s Docs? Here are a handful of article updates that you can tackle:
  - [GraphQL errors, and especially rate limit errors, are not documented](https://github.com/github/docs/issues/22607)
  - [Dismissing a review causes reviewed event to be in dismissed state](https://github.com/github/docs/issues/20216)
  - [Transferring ownership article(s) should contain pointers to how/where transfer request can be approved](https://github.com/github/docs/issues/22565)
- **Are you or do you know a college student interested in open source?** [All In for Students](https://allinopensource.org/access/) is an online program for college students from underrepresented backgrounds that provides open source education, training, and internship opportunities. They’re doing really cool things, check it out!
- **Public APIs:** Ever needed to find a free API for your next project, website, or app? Check out the [Public APIs Repository](https://github.com/public-apis-dev/public-apis)

— 
A very big thank you to everyone in the community who actively contributes thoughts and answers to the questions that come into the API and Webhooks category. Looking forward to seeing more Community contributions to come!

**Comment by Yooo-Its-King** [comment]
GitHubs Webhooks don't work in Discord threads, only specified channels.

**Comment by ghostinhershell** [comment]
Hey Community 👋 

Missed out on our last community check-in? You can check that out here: [API and Webhooks: February Community Check-in 💕](https://github.com/orgs/community/discussions/47680)
